docs/getting-started/security-ui.asciidoc

```diff
@@ -22,204 +23,69 @@ image::images/search-bar.png[]
-* *Manage*: View and manage hosts that are running {endpoint-cloud-sec}.
-
-TIP: Click the *Collapse side navigation*  button to collapse and expand the main navigation menu.
-
-[role="screenshot"]
-image::images/collapse-side-nav-button.gif[width=100%][height=100%][Demo that shows how to to scroll through the page]
-
-[float]
-[[getting-started-page]]
-=== Get started page
-
-The Get started page guides you to integrations that you can use to monitor your hosts and ingest data. Other {security-app} pages that show event data (for example, the Hosts and Network pages) display the get started prompt until event data has been ingested.
-
-image::images/getting-started-pg.png[Shows the Get started page]
-
-[float]
-[[overview-ui]]
-=== Overview dashboard
-
-The Overview dashboard provides a high-level snapshot of detections, external alerts, and event trends. It can help you assess overall system health and find anomalies that may require further investigation. Refer to <<overview-dashboard, Overview dashboard>> for more information.
-
-image::images/overview-pg.png[Overview dashboard]
-
-[float]
-[[detection-response-dashboard-ui]]
-=== Detection & Response dashboard
-
-The Detection & Response dashboard provides focused visibility into the day-to-day operations of your security environment. It helps security operations managers and analysts quickly monitor recent and high priority detection alerts and cases, and identify the hosts and users associated with alerts. Refer to <<detection-response-dashboard, Detection & Response dashboard>> for more information.
-
-[role="screenshot"]
-image::detections/images/detection-response-dashboard.png[Overview of Detection & Response dashboard]
-
-[float]
-[[detection-engine-ui]]
-=== Alerts page
-
-The Alerts page allows you to view and manage all alerts to monitor activity within your network. Refer to <<detection-engine-overview, Detections and Alerts>> for more information.
+<<detection-engine-overview, *Alerts*>>: View and manage alerts to monitor activity within your network.
 
 [role="screenshot"]
 image::detections/images/alert-page.png[]
 
-[float]
-=== Findings page
-
-The Findings page provides an overview of your Kubernetes infrastructure's performance on a range of security benchmarks. Refer to <<kspm, Kubernetes security posture management>>.
+<<findings-page, *Findings*>>: Review your Kubernetes infrastructure’s performance on a range of security benchmarks.
 
 [role="screenshot"]
 image::cloud-native-security/images/findings-page.png[The Findings page]
 
-[float]
-[[rules-page]]
-=== Rules page
-
-The Rules page allows you to view and manage all detection rules. Refer to <<rules-ui-management, Manage detection rules>> for more information about prebuilt and custom rules.
+<<timelines-ui, *Timelines*>>: Investigate alerts and complex threats, such as lateral movement of malware across hosts in your network. Timelines are responsive and allow you to share your findings among other team members.
```

Comment:
```suggestion
<<timelines-ui, *Timelines*>>: Investigate alerts and complex threats, such as lateral movement of malware across hosts in your network. Timelines are interactive and allow you to share your findings among other team members.
```
Just another option 🤷

Reply:
I like "interactive". I'd make the second part of the sentence more active though, for ex:

```suggestion
<<timelines-ui, *Timelines*>>: Investigate alerts and complex threats, such as lateral movement of malware across hosts in your network. Timelines are interactive and you can share your findings with other team members.
```